poetry.lock

```diff
@@ -0,0 +1,3777 @@
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
```

Comment:
@jsolbrig Did we want to include this poetry.lock file?

Reply:
Let me look into this a bit more.

Reply:
Read this, then we should discuss: https://python-poetry.org/docs/basic-usage/#installing-dependencies

Using poetry.lock feels uncomfortable to me. It also makes it difficult for GeoIPS to be installed into an environment that already contains other packages. I'm not sure if I like it or not.

Reply:
Have deleted the poetry.lock file and tried installing using pip; it works as expected. On the sidenote, GeoIPS get installed using poetry install without lock file; it just creates a new one.

Reply:
I have removed the poetry.lock file. Poetry discusses when it should and should not be used. They suggest that it should be used for top-level packages that won't be used as dependencies for other packages. For "libraries" they suggest not using it.

GeoIPS is a little of both, but since other people will be using GeoIPS as a dependency for their plugin packages, it would be best to not use poetry.lock.